openapi/docusign.click/openapi.yaml

```diff
@@ -0,0 +1,1645 @@
+openapi: 3.0.1
+info:
+  title: DocuSign Click API
+  description: DocuSign Click lets you capture consent to standard agreement terms with a single click terms and conditions, terms of service, terms of use, privacy policies, and more. The Click API lets you include this customizable clickwrap solution in your DocuSign integrations.
```

Comment:
Shall we follow the standard used in other connectors in the description as follows
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/ably/openapi.yaml#L14
https://github.com/ballerina-platform/openapi-connectors/blob/49164bb7baae2ddcd19037c109548143c0fe32d7/openapi/spotify/openapi.yaml#L7